src/lib/components/table.svelte

```diff
@@ -87,6 +108,13 @@
                 <Paginator
                     onPageChange={(page) => {
```

Comment:
I am personally an advocate of moving as much logic as possible to the `script` part of the code so it is easier to read, instead of having logic inline with the HTML, but this is a personal preference

Reply:
agree moved it into a function